routes/web/project/location/vm.rb

```diff
@@ -30,9 +30,10 @@ class CloverWeb
             Validation.validate_port_range(r.params["port_range"])
           end
 
+          parsed_cidr = Validation.validate_cidr(r.params["cidr"])
```

Comment:
Although we will be removing resource level firewall-rule endpoints with subsequent PR(s), we can add the same change to api code for the sake of completeness

Reply:
It's already there :) This one was missing.

Reply:
I mean passing `parsed_cidr` to `insert_firewall_rule` on apis as well instead of passing obtained param